denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.example.android.favoritetoys">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
For this simple app I guess this is fine, but consider setting it to `false` as a default moving forward. Handling rtl is pretty easy in most case, but e.g. in our sales app it is not straight forward and therefore set to `false`